vspace/src/main/java/edu/asu/diging/vspace/web/staff/api/SearchSpacesController.java

```diff
@@ -0,0 +1,45 @@
+package edu.asu.diging.vspace.web.staff.api;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import edu.asu.diging.vspace.core.model.ISpace;
+import edu.asu.diging.vspace.core.services.ISpaceManager;
+
+@Controller
+public class SearchSpacesController {
```

Comment:
Is this needed? Can't you use the `StaffSearchSpaceController`?

Reply:
they are actually being used for different tasks

Reply:
What's different about them?

Reply:
?

Reply:
one is mapped to "/staff/spaces/search" and the other is to "/staff/search/space"

Reply:
That doesn't make sense. What's already there should be reused. It's fine if the search searches more than the name.